### PR TITLE
Issue #47 

### PR DESCRIPTION
**Pull request by atuljain**
screenshot from 2015-11-22 23 37 30

### CODE REVIEW

File: coderbounty/settings.py [hunk 1]
```diff
@@ -49,6 +49,12 @@
     'actstream',
     'website',
 )
+SOCIALACCOUNT_PROVIDERS = \
```
Comment (seanauriti) [inline coderbounty/settings.py]: Why is this needed?  It has worked fine without this variable.

File: coderbounty/templates/issue.html [hunk 2]
```diff
@@ -134,12 +248,42 @@ <h2 class="titlebar"><span class="ribbon ribbon-normal ribbon-04 ribbon-brown">T
...
-                    <button class="ribbon ribbon-large ribbon-03">Take</button>
                     <button class="ribbon ribbon-normal ribbon-03 ribbon-brown">Share</button>
                   </form>
                 </div>
             </div>
+            <div class="codeIssueTaker" style="display:none;">
+               <div class="img-holder">
+                    <img src="{% static "images/temp/thumb-medium-00.jpg" %}" alt="" />
+                </div>
+              <div class="user"><strong>Test User</strong></div>
```
Comment (seanauriti) [inline coderbounty/templates/issue.html]: Can you please make this username of the most recent "taker"

File: coderbounty/settings.py (hunk 1 above)
Comment (atuljain) [inline coderbounty/settings.py]: i was testing, i removed this from local and commi latest code

**Comment by atuljain** [comment]
screenshot from 2015-11-23 22 39 34

**Comment by seanauriti** [comment]
@atuljain it has some conflicts, can you please merge and pull again?
